### PR TITLE
Fix #691, improve references

### LINKED ISSUE #691: §References: Issues with the references
|Version|Source|Clause(s)|
|:---:|:---:|:---:|
|AOM Working Group Draft, 17 July 2023|Apple|§References|

#### Comment:
Best Practice is mentioned in a lot of normative references. We are referencing specification. Right? Why is that called Best Practices?

OPUS points to an incomplete spec. Be careful.

Is it possible to avoid referencing Wikipedia?

-----
#### Suggestion:
Improve references

### PR DESCRIPTION
**Pull request by sunghee-hwang**
***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/pull/721.html" title="Last updated on Aug 29, 2023, 2:49 AM UTC (dce5f50)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/721/92797d4...dce5f50.html" title="Last updated on Aug 29, 2023, 2:49 AM UTC (dce5f50)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1302,7 +1253,7 @@ When a value for [=layout_type=] or [=sound_system=] is not supported, parsers c
 
 <dfn noexport>loudness_info()</dfn> provides loudness information for a given audio signal.
 
-All signed values are stored as signed Q7.8 fixed-point values (in [[!Q-Format]]).
+Every signed value is stored as a 16-bit, signed, two’s complement fixed-point value with 8 fractional bits. That is, a 16-bit signed (two’s complement) integer, that is implicitly multiplied by the scaling factor 2^(−8).
```
Comment (sunghee-hwang) [inline index.bs:1256]:
```suggestion
Each signed value is stored as a 16-bit, signed, two’s complement fixed-point value with 8 fractional bits. That is, a 16-bit signed (two’s complement) integer, that is implicitly multiplied by the scaling factor 2^(−8).
```

**Comment by tdaede** [comment]
Let's keep Q format throughout the text, and add a (normative) definition of Q format in the convention section. The definition must cover at least Q7.8 and Q0.8. It can also include an informative reference to Wikipedia if desired.

**Review by podborski — COMMENTED** [review]
Minor suggestion. But It feels like it would be good to use the same style for all the references. For example:

[ISOIEC-23091-3-2018] -> [ISO-CICP]
[ISO14496-14] -> [ISO-MP4]
[ISOBMFF] -> [ISO-BMFF]
[ITU1770-4] -> [ITU-1770-4]
[RFC2119] -> [RFC-2119]

and so on. Basically the [SDO-name] kind of format.

**Comment by tdaede** [comment]
These changes can be either included in this PR or done as a follow up.

File: index.bs [hunk 2]
```diff
@@ -1025,7 +982,7 @@ Bit position : Channel Name
 
 </pre>
 
-<dfn noexport>output_gain</dfn> indicates the gain value to be applied to the mixed channels which are indicated by [=output_gain_flags=], where each mixed channel is generated by downmixing two or more input channels. It is computed as \(20 \times \log_{10}(f)\), where \(f\) is the factor by which to scale the mixed channels. It is stored in a 16-bit, signed, two’s complement fixed-point value with 8 fractional bits (i.e., Q7.8 in [[!Q-Format]]).
+<dfn noexport>output_gain</dfn> indicates the gain value to be applied to the mixed channels which are indicated by [=output_gain_flags=], where each mixed channel is generated by downmixing two or more input channels. It is computed as \(20 \times \log_{10}(f)\), where \(f\) is the factor by which to scale the mixed channels. It is stored as a 16-bit, signed, two’s complement fixed-point value with 8 fractional bits. That is, a 16-bit signed (two’s complement) integer, that is implicitly multiplied by the scaling factor 2^(−8).
```
Comment (sunghee-hwang) [inline index.bs:985]:
```suggestion
<dfn noexport>output_gain</dfn> indicates the gain value to be applied to the mixed channels which are indicated by [=output_gain_flags=], where each mixed channel is generated by downmixing two or more input channels. It is computed as \(20 \times \log_{10}(f)\), where \(f\) is the factor by which to scale the mixed channels. It is stored as a 16-bit, signed, two’s complement fixed-point value with 8 fractional bits (i.e., Q7.8).
```

File: index.bs [hunk 3]
```diff
@@ -1257,7 +1214,7 @@ class MixGainParamDefinition() extends ParamDefinition() {
...
 
-<dfn noexport>default_mix_gain</dfn> specifies the default mix gain value to apply when there are no mix gain parameter blocks with the same [=parameter_block_obu/parameter_id=] provided. This value is expressed in dB and SHALL be applied to all channels in the rendered [=Audio Element=]. It is stored as a 16-bit, signed, two's complement fixed-point value with 8 fractional bits (i.e., Q7.8 in [[!Q-Format]]).
+<dfn noexport>default_mix_gain</dfn> specifies the default mix gain value to apply when there are no mix gain parameter blocks with the same [=parameter_block_obu/parameter_id=] provided. This value is expressed in dB and SHALL be applied to all channels in the rendered [=Audio Element=]. It is stored as a 16-bit, signed, two’s complement fixed-point value with 8 fractional bits. That is, a 16-bit signed (two’s complement) integer, that is implicitly multiplied by the scaling factor 2^(−8).
```
Comment (sunghee-hwang) [inline index.bs:1217]:
```suggestion
<dfn noexport>default_mix_gain</dfn> specifies the default mix gain value to apply when there are no mix gain parameter blocks with the same [=parameter_block_obu/parameter_id=] provided. This value is expressed in dB and SHALL be applied to all channels in the rendered [=Audio Element=]. It is stored as a 16-bit, signed, two’s complement fixed-point value with 8 fractional bits (i.e., Q7.8).
```

File: index.bs [hunk 4]
```diff
@@ -1336,7 +1293,7 @@ When a value for [=layout_type=] or [=sound_system=] is not supported, parsers c
 
 <dfn noexport>LoudnessInfo()</dfn> provides loudness information for a given audio signal.
 
-All signed values are stored as signed Q7.8 fixed-point values (in [[!Q-Format]]).
+Each signed value is stored as a 16-bit, signed, two’s complement fixed-point value with 8 fractional bits. That is, a 16-bit signed (two’s complement) integer, that is implicitly multiplied by the scaling factor 2^(−8).
```
Comment (sunghee-hwang) [inline index.bs:1296]:
```suggestion
Each signed value is stored as Q7.8 fixed-point values.
```

File: index.bs [hunk 5]
```diff
@@ -1504,7 +1461,7 @@ animation_type : Animation Type
        2       : BEZIER
 </pre>
 
-<dfn noexport>param_data</dfn> uses the AnimatedParameterData function template. Each of the values defined within this instance ([=start_point_value=], [=end_point_value=], and [=control_point_value=]) is expressed in dB. The values SHALL be applied to all channels in the rendered [=Audio Element=] and SHALL be applied as described in [[#processing-animated-params]]. They are stored as 16-bit, signed, two's complement fixed-point values with 8 fractional bits (i.e., Q7.8 in [[!Q-Format]]).
+<dfn noexport>param_data</dfn> uses the AnimatedParameterData function template. Each of the values defined within this instance ([=start_point_value=], [=end_point_value=], and [=control_point_value=]) is expressed in dB. The values SHALL be applied to all channels in the rendered [=Audio Element=] and SHALL be applied as described in [[#processing-animated-params]]. They are stored as 16-bit, signed, two's complement fixed-point values with 8 fractional bits. That is, a 16-bit signed (two’s complement) integer, that is implicitly multiplied by the scaling factor 2^(−8).
```
Comment (sunghee-hwang) [inline index.bs:1464]:
```suggestion
<dfn noexport>param_data</dfn> uses the AnimatedParameterData function template. Each of the values defined within this instance ([=start_point_value=], [=end_point_value=], and [=control_point_value=]) is expressed in dB. The values SHALL be applied to all channels in the rendered [=Audio Element=] and SHALL be applied as described in [[#processing-animated-params]]. They are stored as 16-bit, signed, two's complement fixed-point values with 8 fractional bits (i.e., Q7.8).
```

**Comment by sunghee-hwang** [comment]
It is merged as it resolves all comments from SG.

**Pull request merged**